config/locales/candidate_interface/degree_empty_component.yml

```diff
@@ -0,0 +1,11 @@
+en:
+  candidate_interface:
+    degree_empty_component:
+      undergraduate:
+        name: Teacher degree apprenticeships
+        bachelor_degree: You do not need a bachelor’s degree to do a %{link}.
```

Comment:
rather than doing `html_safe` in the erb file, we can do `bachelor_degree_html`. Not a blocker. I prefer to have it here, just keeping the `erbs` a bit cleaner, but happy to do it the other way as well.

Reply:
Agree with this

Reply:
Ok, done ;)